amundsen_application/static/js/features/ColumnList/ColumnLineage/index.tsx

```diff
@@ -42,6 +43,18 @@ const getLink = (table) => {
   return `/table_detail/${cluster}/${database}/${schema}/${name}?source=column_lineage`;
 };
 
+const onLineageClick = (e) => {
+  logClick(e, {
```

Comment:
Nit: On the internal handlers, we use the "handle" prefix, while on the component API we use 'on' as the prefix. You can check more in https://github.com/amundsen-io/amundsenfrontendlibrary/blob/master/docs/react_naming_conventions.md#handler-functions